simulator/mips/mips_instr.h

```diff
@@ -189,6 +189,10 @@ class BaseMIPSInstr
                 v_src2 = value;
         }
 
+        void set_new_PC( Addr value) { new_PC = value; }
```

Comment:
If I understand correctly, this method is needed only for re-use of instruction within a single test case.
If you remove re-uses, it won't be necessary anymore.